docs/docs/getting-started/installation.md

```diff
@@ -83,13 +83,50 @@ There is also an [React Native VSCode extension](https://marketplace.visualstudi
 
 ## Testing with Jest
```

Comment:
@SimenB this is what I going with, hope it makes sense and a big thank you for your valuable inputs.